<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>495</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_selected">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell Extra Bold</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Selected: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_formula">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>321</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/maxresdefault.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell Extra Bold</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>What is your skill?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>370</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell Extra Bold</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell Extra Bold</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Modern No. 20</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Beginner</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Modern No. 20</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Intermediate</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Modern No. 20</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Advanced</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
